go-controller/pkg/util/node_annotations.go

```diff
@@ -62,6 +63,21 @@ const (
 	// OvnNodeManagementPort is the constant string representing the annotation key
 	OvnNodeManagementPort = "k8s.ovn.org/node-mgmt-port"
 
+	// OvnNodeManagementPortInfo contains all ips, mac addresses and mgmt ports
+	// on all networks keyed by the network-name
+	// k8s.ovn.org/node-mgmt-port-info: {
+	// 		"default": {
+	// 			"ip-addresses": [ "10.192.13.2/26" ],
+	// 			"mac-address": "ca:53:88:23:bc:98",
+	// 			"function-info": { "PfId": 0, "FuncId": 0 }
+	//      },
+	// 		"l2-network": {
+	// 			"ip-addresses": [ "10.200.10.2/26" ],
+	// 			"mac-address": "5e:52:2a:c0:98:f4"
+	//      }
```

Comment:
Note: for this PR, only the ip-addresses attribute is being implemented. We chose to defer handling the other two as they are currently redundant. Also, `mac addresses` (plural) is new so it is a bit of a churn to change that yet again.